regression-test/data/inverted_index_p2/test_variant_index_format_v1.out

```diff
@@ -1,10 +0,0 @@
--- This file is automatically generated. You should know what you did if you want to edit this
```

Comment:
Is it ok  to remove those test? what if users already use V1 as format? maybe we need to keep those tests to make sure there is no severe bugs